components/makeSubmit.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+import React from "react";
+import { SubmitProps, submitPropTypes } from "remultiform/step";
+import { Button } from "lbh-frontend-react/components/Button";
+
+export const makeSubmit = ({
+  href,
+  value
+}: {
+  href: string;
+  value: string;
+}): React.FunctionComponent<SubmitProps> => {
+  const Submit = ({ onSubmit }: SubmitProps): JSX.Element => {
+    return (
+      <Link href={href}>
+        <Button onClick={() => onSubmit()}>{value}</Button>
```

Comment:
We need to do something if `onSubmit` rejects. Doing this means we never find out about errors. Maybe it's ok to `.catch(console.error)`.